### ISSUE TITLE
feature requests: support remote session url rpc path (transmission qt client)

### ISSUE THREAD

**Issue by uprprc777** [transmission/transmission#5886]
### What is the issue?

For Qt clients (Windows, Linux, BSD, etc.), I can't find any option to specify the HTTP RPC URL path while only host name, port, and authentification.
I have my transmission daemon setup behind a nginx server and have a custom HTTP RPC URL path (because I need to have multiple daemon under one hostname).
As a result, I am not able to connect my daemon via the transmission Qt Client.

For now, I need to use a third party client to access.
But I think the official app should have this feature.

### Which application of Transmission?

Qt app on Windows / Linux

### Which version of Transmission?

4.0.3 (6b0e49bbb2)

**Comment by killemov** [comment]
To be clear, you have differentiated several transmission-daemons by path, not by ip, port, or subdomain. Could you make that differentiation with nginx? You could even differentiate on client ip, but that would be very ghetto.

**Comment by uprprc777** [comment]
> To be clear, you have differentiated several transmission-daemons by path, not by ip, port, or subdomain. Could you make that differentiation with nginx? You could even differentiate on client ip, but that would be very ghetto.
> 
> Also see #5865

I have sth like `https://transmission.mydomain.com/transmission-instance-01/rpc` with nginx reverse proxy to `127.0.0.1:50001`, `https://transmission.mydomain.com/transmission-instance-02/rpc` with nginx reverse proxy to `127.0.0.1:50002`...

The transmission daemons are bind to some different ports of localhost.
I have disabled direct ip:port access with the firewall on the server. So to access those transmission-daemon can only through urls.

I have my transmission and nginx setup successfully and have no trounle with a browser or some third party transmission clients.
But I think it would be great for the official app to have this feature.

**Comment by Chaz6** [comment]
I don't know if it is helpful or not, but the client [Tremotesf](https://github.com/equeim/tremotesf2) allows you to specify the path.

![image](https://github.com/transmission/transmission/assets/12680/8d5c59f7-40ce-4542-ba7d-6cda04ecc226)

**Comment by uprprc777** [comment]
> I don't know if it is helpful or not, but the client [Tremotesf](https://github.com/equeim/tremotesf2) allows you to specify the path.
> 
> ![image](https://user-images.githubusercontent.com/12680/260232646-8d5c59f7-40ce-4542-ba7d-6cda04ecc226.png)

Yes, I am actually using this client app. But I hope if the official app can have this feature.
